src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/BookingFacadeTest.kt

```diff
@@ -578,6 +579,60 @@ class BookingFacadeTest {
       }
     }
 
+    @Test
+    fun `should send emails but no events on cancellation of a court booking by service user`() {
+      contactsService.stub {
```

Comment:
Will this always be true that calling cancel via a ServiceUser should avoid events? Wonder if other circumstances might crop up were an external trigger might prompt a cancel and we do want events emitted? Just a thought, that something more explicit might be safer? Doesn't happen today anyway.

Reply:
In truth I am a little concerned the facade is getting too big for its boots in terms of responsibilities 🤷 

Food for thought.